avalon/pipeline.py

```diff
@@ -248,7 +248,7 @@ def process(self, session, **kwargs):
         pass
 
 
-class InventoryAction(Action):
+class InventoryAction(object):
     """A custom action for tools, e.g: sceneinventory"""
```

Comment:
The docstring can use a slight improvement, this is not for "tools" in general, but specifically to those tailored at the scene's inventory.